src/components/cart-product-card/Cart-Product-Card.js

```diff
@@ -0,0 +1,79 @@
+import "./Cart-Product-Card.css";
+import {
+  axiosAddQty,
+  axiosDecrementQty,
+  axiosRemoveFromCart
+} from "../../utilty/cart-utility";
+import { axiosAddToWishlist } from "../../utilty/wishlist-utility";
+
+import { useCart } from "../../contexts/Cart-Context";
+import { useWishlist } from "../../contexts/Wishlist-Context";
+
+export function CartProductCard({ prd }) {
+  const { cartState, cartDispatch } = useCart();
+  const { wishlistItems, setwishlistItems } = useWishlist();
+
+  function getProdTotal(prod) {
+    const total = parseInt(prod.discountedPrice) * parseInt(prod.qty);
+    return total;
+  }
+
+  return (
+    <div key={prd._id} className="horizCardParent">
+      <div className="horizCardBody">
+        <div className="HorizImgDiv">
+          <img src={prd.image} />
+        </div>
+        <div className="HorizCardDetails">
+          <div className="brandTitle">{prd.name}</div>
+          <p className="prdDescrip">{prd.brand}</p>
+          <div className="horizCardBtnDiv">
+            <button
+              className="HorizQtyBtn"
+              onClick={() => {
+                axiosAddQty(prd, cartDispatch);
+                // cartDispatch({ type: "INCREMENT", payload: prd });
+              }}
+            >
+              +
+            </button>
+            <span className="amt"> {prd.qty} </span>
+            <button
+              className="HorizQtyBtn subBtn"
+              onClick={() => {
+                axiosDecrementQty(prd, cartDispatch);
+              }}
+            >
+              -
+            </button>
+          </div>
+          <div className="prdPrice">
+            <b> ₹{getProdTotal(prd)}</b>
+          </div>
+        </div>
+      </div>
+      <div className="horizCardFooter">
+        <button
+          className="horizFooterBtn secBtn"
+          onClick={() => {
+            axiosRemoveFromCart(prd, cartDispatch);
+          }}
+        >
+          Remove
+        </button>
+        <button
+          onClick={() => {
+            // setwishlistItems((prds) => [...prds, prd]);
+            // removeFromCart(prd);
+            axiosAddToWishlist(prd, wishlistItems, setwishlistItems);
+            axiosRemoveFromCart(prd, cartDispatch);
+            // cartDispatch({ type: "REMOVE", payload: prd });
+          }}
+          className="horizFooterBtn"

```

Comment:
1. currently in the UI,\
after hovering over *wishlist* button,\
only the background color changes\
but after clicking the button - even though the item gets added to wishlist in backend\
but the heart icon on the wishlist does not gets filled after the wishlist item gets added.

2. on hovering over buttons\
add a *cursor: pointer* class just to keep accessibility in mind.